### ISSUE TITLE
Need high-res icons for Mac

### ISSUE THREAD

**Issue by MichaelBuckley** [snes9xgit/snes9x#551]
For as long as I can remember, on Mac, Snes9x has used a SNES or Super Famicom for its icon, as opposed to the project logo used on other platforms. When Mac OS X came out, the port got new icons that apear to be 3D renders of a Super Famicom and its cartridges. However, the source douments used to create these icons are not in the repository.

These icons come in 128x128 and 32x32 sizes. However, on modern versions of macOS, applications are supposed to supply icons at the following wises: 16x16, 32x32, 128x128, 256x256, 512x512, and 1024x1024. The largest size is for high-density monitors.

There's nothing stopping us from continuting to use the current icons, aside from the fact that they will look blurry on high-density monitors and when users size their icons larger.

I could also reach out to some of my contacts to see if they would be willing to design new icons, but I figured that it would check with the maintainers to see if they had any thoughts or preferences first.

Currently, the port uses the following icons.

Application:
![APPL](https://user-images.githubusercontent.com/50380/61188660-72be5880-a637-11e9-9240-c12ee0aefd81.png)

ROM file:
![CART](https://user-images.githubusercontent.com/50380/61188666-8b2e7300-a637-11e9-9fb7-c8920689543a.png)

SRAMs folder:
![folder_SRAMs](https://user-images.githubusercontent.com/50380/61188683-c9c42d80-a637-11e9-84f6-00f4080b59d0.png)

SRAM file:
![SRAM](https://user-images.githubusercontent.com/50380/61188691-df395780-a637-11e9-8564-479e3ace49ce.png)

Freezes folder:
![folder_Freezes](https://user-images.githubusercontent.com/50380/61188719-30494b80-a638-11e9-897a-ae3360e70e79.png)

Freeze file:
![SAVE](https://user-images.githubusercontent.com/50380/61188730-5242ce00-a638-11e9-870f-713ea2ffdcf5.png)

**Comment by eisnerguy1** [comment]
I was just wondering if there's any updates on the new macOS icons?

**Comment by MichaelBuckley** [comment]
Nope. No one has stepped forward to volunteer, and I haven't commissioned anyone. I gave a try at createing a Mavericks-style icon, but the icons in the preferences are about hte best I can do with my artistic skill, and it doesn't look good at 1024x1024.

(I'm not precious about the preference icons either. If someone comes up with better icons, I'm happy to swap them in.)
